projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```diff
@@ -769,7 +785,7 @@ textarea#jp-carousel-comment-form-comment-field:focus::-webkit-input-placeholder
 
 #jp-carousel-comment-form-commenting-as input {
 	background: rgba( 34, 34, 34, 0.9 );
```

Comment:
In light theme I see:
<img width="1108" alt="Screen Shot 2021-08-02 at 1 49 40 pm" src="https://user-images.githubusercontent.com/6458278/127802195-b030d54c-221d-4d16-969a-67d253e7a3f4.png">

```suggestion
	background: var(--jp-carousel-bg-color);
```

Reply:
Also, I think it might be good to update the focus color background on [this line](https://github.com/Automattic/jetpack/pull/20503/files#diff-72023849d8cb93014b6aafc178f8b48eadb46607ad5c9eb6b2a0c419deebf243R802).

```css
#jp-carousel-comment-form-commenting-as input:focus {
	background: var(--jp-carousel-bg-faded-color);
	color: #222;
}
```

Reply:
Good suggestions! Updated in https://github.com/Automattic/jetpack/pull/20503/commits/4a4b52b89c6c6e14def2e60b366484b640fa20b2